sorry.html

```diff
@@ -3,15 +3,15 @@
 <head>
     <base href="https://sorry.thebiggive.org.uk">
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <link href="https://cdn.jsdelivr.net/npm/@biggive/components/dist/biggive/assets/fonts/EuclidTriangle/stylesheet.css" rel="stylesheet" type="text/css" />
-    <script type="module" src="https://cdn.jsdelivr.net/npm/@biggive/components/dist/biggive/biggive.esm.js" defer></script>
-    <script nomodule src="https://cdn.jsdelivr.net/npm/@biggive/components/dist/components/index.js" defer></script>
+    <link href="https://cdn.jsdelivr.net/npm/@biggive/components@202311241148.0.0/dist/biggive/assets/fonts/EuclidTriangle/stylesheet.css" rel="stylesheet" type="text/css" />
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@biggive/components@202311241148.0.0/dist/biggive/biggive.esm.js" defer></script>
+    <script nomodule src="https://cdn.jsdelivr.net/npm/@biggive/components@202311241148.0.0/dist/components/index.js" defer></script>
```

Comment:
:+1: to pinning, we don't know if later components versions will be compatible or will have a new required param for the footer or something. I do think if we're pinning then at some point in future we probably might as well bundle a copy of the components into `sorrry` - could be as a build step or it would probably be fine to just have it saved into this repo.

Reply:
Possibly – I do wonder if it's even worth this having CI since we change it so rarely, but I guess it's a bit circular – we would possibly tune it more if it was easier to deploy.

Reply:
Yeah we'll never want to be changing this often, I think I'd probably just put a copy of the built components library inside this git repo.